src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class LimelightSubsystem extends SubsystemBase {
+  /** Creates a new LimelightSubsystem. */
+  public LimelightSubsystem() {}
+  
+  // current LedMode, 0= on, 1 = off
+  private int CurMode = 0;
+  
+  NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+
+  @Override
+  public void periodic() {
+    NetworkTableEntry ty = table.getEntry("ty");
+    double y = ty.getDouble(0);
+    SmartDashboard.putNumber("LimelightY", y);
```

Comment:
All this is basically the getTy() code you have below.  If you need the ty value then why not simply call getTy() here to avoid duplicate code?